app.py

```diff
@@ -174,6 +187,29 @@ def main():
             capture_exception(e_exception)
             return jsonify({"status": False, "debugMessage": "Request Failed", "error": str(e_exception), "start_time": start_time, "total_time_taken": total_time.seconds, 'version': version, 'code': '2'})
 
+    if request.args.get('method') == "advocatecasesbynumber":
```

Comment:
does it handle invalid input, for example if case not found incase they pass a wrong case number or year.